test/unit/plugins/rest_spec.js

```diff
@@ -134,5 +139,237 @@ describe("The Rest plugin", function () {
 				expect(result.statusCode).to.equal(500);
 			});
 		});
+		describe("with an invalid instance", function () {
```

Comment:
Our `describe` blocks are starting to get a little disorganized. Making the test output readable is helpful for debugging when things to wrong. I think our test output should look a little more like this:

```
The Rest plugin
    creating a new instance
        with valid parameters
        [ ... ]
    getting an instance
        with valid parameters
        [ ... ]
    querying for instances
        [ ... ]
```

This organizes the tests into specific, descriptive sections. You should be able to accomplish this by rearranging and adding `describe` blocks.